plugins/clients/apps-client/test/unit/test_apps_client.py

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
Appart from the tests, add examples on how to use multiplr response types on `client_example` application.